config/config.go

```diff
@@ -248,6 +251,8 @@ type (
 		EnableSystemLogIndexer bool `yaml:"enableSystemLog"`
 		// EnableStakingProtocol enables staking protocol
 		EnableStakingProtocol bool `yaml: "enableStakingProtocol"`
+		// EnableStakingIndexer enables staking indexer
+		EnableStakingIndexer bool `yaml: "enableStakingIndexer"`
```

Comment:
`yaml: "enableStakingIndexer"` remove the space between `:` and `"`

also plz do it for
```
EnableStakingProtocol bool `yaml: "enableStakingProtocol"`
``` 
as well

Reply:
ok